Rocket.toml

```diff
@@ -0,0 +1,26 @@
+[development]
+address = "0.0.0.0"
+port = 8000
+workers = 2
+keep_alive = 5
+read_timeout = 5
+write_timeout = 5
+log = "normal"
+
+[staging]
+address = "0.0.0.0"
+port = 8000
+workers = 4
+keep_alive = 5
+read_timeout = 5
+write_timeout = 5
+log = "normal"
+
+[production]
+address = "0.0.0.0"
+port = 8000
+workers = 8
+keep_alive = 5
+read_timeout = 10
+write_timeout = 10
+log = "critical"
```

Comment:
Unless I'm misinterpreting the docs, `Rocket.toml` is for **runtime** overwrites. Meaning the file has to be present at runtime for it to work. We don't really want that, as it's intended for the end user to configure the web server.

Reply:
It's not required to be present, it'll use the rocket defaults when missing

Reply:
Well yeah, but that means sacrificing what you're trying to implement when exporting outside of a development environment.

If it's just for the address and port fix, it might be worth not comitting this file - that way the behaviour for anyone that pulls & builds this repository stays the same regardless of configuration (as they won't pull in a `Rocket.toml`)

Reply:
I don't get what is being sacrificed here. It sits in the same directory as the executable as a sort of config file